[WebExtensions] Document runtime.getVersion() method (#42971)
mdn/content: files/en-us/mozilla/add-ons/webextensions/api/runtime/getmanifest/index.md

sidebar: addonsidebar
---

> Get the object representation of [manifest.json](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json) file. This representation is produced from browser-internal data structures and may differ from what would be produced by running `JSON.parse()` on the actual file within extension.

Reviewer: ```suggestion
Get the extension's [`manifest.json`](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json) file as an object. The object is created from browser-internal data structures and can differ from a representation produced by running `JSON.parse()` on the file in the extension.
```
